[31m[Nest] 72  - [39m12/01/2023, 4:40:34 PM [31m  ERROR[39m [38;5;3m[ExceptionHandler] [39m[31mNest can't resolve dependencies of the UserService (?). Please make sure that the argument "UserEntityRepository" at index [0] is available in the UserModule context.

Potential solutions:
- Is UserModule a valid NestJS module?
- If "UserEntityRepository" is a provider, is it part of the current UserModule?
- If "UserEntityRepository" is exported from a separate @Module, is that module imported within UserModule?
  @Module({
    imports: [ /* the Module containing "UserEntityRepository" */ ]
  })
[39m
Error: Nest can't resolve dependencies of the UserService (?). Please make sure that the argument "UserEntityRepository" at index [0] is available in the UserModule context.

Potential solutions:
- Is UserModule a valid NestJS module?
- If "UserEntityRepository" is a provider, is it part of the current UserModule?
- If "UserEntityRepository" is exported from a separate @Module, is that module imported within UserModule?
  @Module({
    imports: [ /* the Module containing "UserEntityRepository" */ ]
  })

    at Injector.lookupComponentInParentModules (/api/node_modules/@nestjs/core/injector/injector.js:254:19)
    at Injector.resolveComponentInstance (/api/node_modules/@nestjs/core/injector/injector.js:207:33)
    at resolveParam (/api/node_modules/@nestjs/core/injector/injector.js:128:38)
    at async Promise.all (index 0)
    at Injector.resolveConstructorParams (/api/node_modules/@nestjs/core/injector/injector.js:143:27)
    at Injector.loadInstance (/api/node_modules/@nestjs/core/injector/injector.js:70:13)
    at Injector.loadProvider (/api/node_modules/@nestjs/core/injector/injector.js:97:9)
    at /api/node_modules/@nestjs/core/injector/instance-loader.js:56:13
    at async Promise.all (index 3)
    at InstanceLoader.createInstancesOfProviders (/api/node_modules/@nestjs/core/injector/instance-loader.js:55:9)
[31m[Nest] 319  - [39m12/01/2023, 4:42:19 PM [31m  ERROR[39m [38;5;3m[ExceptionHandler] [39m[31mNest can't resolve dependencies of the UserService (?). Please make sure that the argument "UserEntityRepository" at index [0] is available in the UserModule context.

Potential solutions:
- Is UserModule a valid NestJS module?
- If "UserEntityRepository" is a provider, is it part of the current UserModule?
- If "UserEntityRepository" is exported from a separate @Module, is that module imported within UserModule?
  @Module({
    imports: [ /* the Module containing "UserEntityRepository" */ ]
  })
[39m
Error: Nest can't resolve dependencies of the UserService (?). Please make sure that the argument "UserEntityRepository" at index [0] is available in the UserModule context.

Potential solutions:
- Is UserModule a valid NestJS module?
- If "UserEntityRepository" is a provider, is it part of the current UserModule?
- If "UserEntityRepository" is exported from a separate @Module, is that module imported within UserModule?
  @Module({
    imports: [ /* the Module containing "UserEntityRepository" */ ]
  })

    at Injector.lookupComponentInParentModules (/api/node_modules/@nestjs/core/injector/injector.js:254:19)
    at Injector.resolveComponentInstance (/api/node_modules/@nestjs/core/injector/injector.js:207:33)
    at resolveParam (/api/node_modules/@nestjs/core/injector/injector.js:128:38)
    at async Promise.all (index 0)
    at Injector.resolveConstructorParams (/api/node_modules/@nestjs/core/injector/injector.js:143:27)
    at Injector.loadInstance (/api/node_modules/@nestjs/core/injector/injector.js:70:13)
    at Injector.loadProvider (/api/node_modules/@nestjs/core/injector/injector.js:97:9)
    at /api/node_modules/@nestjs/core/injector/instance-loader.js:56:13
    at async Promise.all (index 3)
    at InstanceLoader.createInstancesOfProviders (/api/node_modules/@nestjs/core/injector/instance-loader.js:55:9)
[31m[Nest] 341  - [39m12/01/2023, 4:42:22 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (1)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:25 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (2)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:28 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (3)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:31 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (4)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:34 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (5)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:37 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (6)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:40 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (7)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:43 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (8)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:46 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (9)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 341  - [39m12/01/2023, 4:42:46 PM [31m  ERROR[39m [38;5;3m[ExceptionHandler] [39m[31mEntity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:42:52 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (1)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:42:55 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (2)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:42:58 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (3)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:01 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (4)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:04 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (5)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:07 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (6)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:10 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (7)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:13 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (8)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:16 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (9)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 422  - [39m12/01/2023, 4:43:16 PM [31m  ERROR[39m [38;5;3m[ExceptionHandler] [39m[31mEntity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 509  - [39m12/01/2023, 4:43:28 PM [31m  ERROR[39m [38;5;3m[TypeOrmModule] [39m[31mUnable to connect to the database. Retrying (1)...[39m
MissingPrimaryColumnError: Entity "UserEntity" does not have a primary column. Primary column is required to have in all your entities. Use @PrimaryColumn decorator to add a primary column to your entity.
    at EntityMetadataValidator.validate (/api/src/metadata-builder/EntityMetadataValidator.ts:59:19)
    at /api/src/metadata-builder/EntityMetadataValidator.ts:43:18
    at Array.forEach (<anonymous>)
    at EntityMetadataValidator.validateMany (/api/src/metadata-builder/EntityMetadataValidator.ts:42:25)
    at DataSource.buildMetadatas (/api/src/data-source/DataSource.ts:724:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at DataSource.initialize (/api/src/data-source/DataSource.ts:259:13)
[31m[Nest] 534  - [39m12/01/2023, 4:43:46 PM [31m  ERROR[39m [38;5;3m[ExceptionsHandler] [39m[31mrelation "user_entity" does not exist[39m
QueryFailedError: relation "user_entity" does not exist
    at PostgresQueryRunner.query (/api/src/driver/postgres/PostgresQueryRunner.ts:299:19)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at SelectQueryBuilder.loadRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3789:25)
    at SelectQueryBuilder.executeEntitiesAndRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3541:26)
    at SelectQueryBuilder.getRawAndEntities (/api/src/query-builder/SelectQueryBuilder.ts:1670:29)
    at SelectQueryBuilder.getMany (/api/src/query-builder/SelectQueryBuilder.ts:1760:25)
    at /api/node_modules/@nestjs/core/router/router-execution-context.js:46:28
    at /api/node_modules/@nestjs/core/router/router-proxy.js:9:17
[31m[Nest] 534  - [39m12/01/2023, 4:43:57 PM [31m  ERROR[39m [38;5;3m[ExceptionsHandler] [39m[31mrelation "user_entity" does not exist[39m
QueryFailedError: relation "user_entity" does not exist
    at PostgresQueryRunner.query (/api/src/driver/postgres/PostgresQueryRunner.ts:299:19)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at SelectQueryBuilder.loadRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3789:25)
    at SelectQueryBuilder.executeEntitiesAndRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3541:26)
    at SelectQueryBuilder.getRawAndEntities (/api/src/query-builder/SelectQueryBuilder.ts:1670:29)
    at SelectQueryBuilder.getMany (/api/src/query-builder/SelectQueryBuilder.ts:1760:25)
    at /api/node_modules/@nestjs/core/router/router-execution-context.js:46:28
    at /api/node_modules/@nestjs/core/router/router-proxy.js:9:17
[31m[Nest] 534  - [39m12/01/2023, 4:46:45 PM [31m  ERROR[39m [38;5;3m[ExceptionsHandler] [39m[31mrelation "user_entity" does not exist[39m
QueryFailedError: relation "user_entity" does not exist
    at PostgresQueryRunner.query (/api/src/driver/postgres/PostgresQueryRunner.ts:299:19)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at SelectQueryBuilder.loadRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3789:25)
    at SelectQueryBuilder.executeEntitiesAndRawResults (/api/src/query-builder/SelectQueryBuilder.ts:3541:26)
    at SelectQueryBuilder.getRawAndEntities (/api/src/query-builder/SelectQueryBuilder.ts:1670:29)
    at SelectQueryBuilder.getMany (/api/src/query-builder/SelectQueryBuilder.ts:1760:25)
    at /api/node_modules/@nestjs/core/router/router-execution-context.js:46:28
    at /api/node_modules/@nestjs/core/router/router-proxy.js:9:17
